src/lib/bounded-contexts/marketing/marketing/tests/__tests__/todo-completions-incremented/stream-command-bus.mock.ts

```diff
@@ -0,0 +1,26 @@
+import { Infra, ok } from '@bitloops/bl-boilerplate-core';
+import { SendEmailCommand } from '../../../commands/send-email.command';
+
+export class MockStreamCommandBus {
+  public readonly mockPublish: jest.Mock;
+  private mockStreamCommandBus: Infra.CommandBus.IStreamCommandBus;
+
+  constructor() {
+    this.mockPublish = this.getMockPublishMethod();
+    this.mockStreamCommandBus = {
+      publish: this.mockPublish,
+      subscribe: jest.fn(),
+    };
+  }
+
+  getMockStreamingCommandBus(): Infra.CommandBus.IStreamCommandBus {
+    return this.mockStreamCommandBus;
+  }
+
+  private getMockPublishMethod(): jest.Mock {
+    return jest.fn((command: SendEmailCommand) => {
+      console.log('Publishing...', command);
+      return Promise.resolve(ok());
+    });
+  }
+}
```

Comment:
This file is repeated in every mocking of command bus. Move it in the marketing `tests/mocks` folder in order for everyone to reuse it. If in any case someone has to mock it in a different way then it would implement it in it's folder.